user/schemas.py

```diff
@@ -1,8 +1,15 @@
 from pydantic import BaseModel
 
 
-class Userdetail(BaseModel):
-    name : str 
-    email : str 
-    password : str
+class User(BaseModel):
+    name: str 
+    email: str
+    password: str
+
+class ShowUser(BaseModel):
```

Comment:
I hope this schema class is used for UpdateUser. Kindly check and update the class name here as well as on the used places.